esileapclient/osc/v1/node.py

```diff
@@ -48,20 +49,36 @@ def get_parser(self, prog_name):
             dest='lessee',
             required=False,
             help="Filter nodes by lessee.")
+        parser.add_argument(
+            '--property',
+            dest='properties',
+            required=False,
+            action='append',
+            help="Filter nodes by properties. Format: 'key>=value'. "
+                 "Supported operators: >=, <=, >, <, =."
+                 "Can be specified multiple times.",
```

Comment:
We can ensure this is always in sync with the code in `utils` by dynamically generating the help output:

```suggestion
            help="Filter nodes by properties. Format: 'key>=value'. "
                 "Can be specified multiple times. "
                 f"Supported operators are: {', '.join(utils.OPS}",
```

Reply:
Done. I will do it.